.travis.yml

```diff
@@ -27,6 +27,7 @@ addons:
         packages:
             - cmake
             - build-essential
+            - ninja
```

Comment:
I think you might want to use [ninja-build](https://packages.ubuntu.com/trusty/ninja-build) instead of [ninja](https://packages.ubuntu.com/trusty/ninja) here. At least according to https://github.com/GenericMappingTools/gmt/pull/353/files#diff-354f30a63fb0907d4ad57269548329e3R102

Reply:
Oops, should have been clearer. Use `ninja-build` just for linux, and keep using `ninja` for homebrew.

Reply:
OK. I've made the changes. I never used ninja before.

Reply:
Neither have I, not sure what it does :rofl: